GitUtils.ps1

```diff
@@ -256,8 +256,7 @@ function Get-SshAgent() {
     } else {
         $agentPid = $Env:SSH_AGENT_PID
         if ($agentPid) {
-            $sshAgentProcess = Get-Process | Where-Object { $_.Id -eq $agentPid -and $_.Name -eq 'ssh-agent' }
-            if ($null -ne $sshAgentProcess) {
+            if (((Get-Process -Id $agentPid -FileVersionInfo -ErrorAction SilentlyContinue).Filename).Contains("Git")) {

```

Comment:
If `$agentPid` is valid but the process has been killed, this will blow up when you try to access `Filename`.